src/core/task8.js

```diff
@@ -1,4 +1,4 @@
-export function createArray(number) {
```

Comment:
What will be if we pass a negative value? For example, -5.
The function should return 0, -1, -2, -3, -4, -5 in this case.